api/src/certification/results/infrastructure/utils/pdf/templates/certificate.js

```diff
@@ -189,8 +178,241 @@ export default function generateV3AttestationTemplate({ pdf, data, translate })
+  pdf
+    .font('Roboto-Regular')
+    .fontSize(10)
+    .fillColor('#253858')
+    .text(translate('certification.attestation.v2.signature'), 345, 602);
+
+  pdf
+    .font('Roboto-Regular')
+    .fontSize(8)
+    .fillColor('#253858')
+    .text(translate('certification.attestation.v2.not-a-certificate'), 0, pdf.page.height - 17, {
+      width: pdf.page.width,
+      height: 10,
+      align: 'center',
+    });
+
+  // complementary certification content
+  if (data.certifiedBadges.length > 0) {
+    pdf
+      .font('Roboto-Regular')
+      .fontSize(10)
+      .fillColor('#253858')
+      .text(translate('certification.attestation.v2.complementary-certification-title'), 345, 338);
+
+    pdf.roundedRect(343, 355, 197, 160, 5).fillOpacity(0.8).fill('#ffffff');
+
+    const badge = data.certifiedBadges[0].stickerUrl.slice(39, -4);
+    pdf.image(path.resolve(__badgesDirname, `${badge}.png`), 343, 373, {
+      fit: [197, 70],
+      align: 'center',
+      valign: 'center',
+    });
+
+    pdf
+      .font('Roboto-Regular')
+      .fontSize(8)
+      .fillColor('#253858')
+      .text(data.certifiedBadges[0].message, 351, 452, { width: 190 });
+  }
+
+  // competences content
+  pdf
+    .font('Roboto-Regular')
+    .fontSize(10)
+    .fillColor('#253858')
+    .text(
+      translate('certification.attestation.v2.competences-content.title', {
+        maxReachableLevelOnCertificationDate: data.maxReachableLevelOnCertificationDate,
+      }),
+      55,
+      225,
+    );
+
+  [247, 352, 480, 608, 712].forEach((y, index) => {
+    pdf
+      .font('Roboto-Regular')
+      .fontSize(9)
+      .fillColor('#253858')
+      .text(data.resultCompetenceTree.areas[index].title, 65, y);
+
+    pdf.text(translate('certification.attestation.v2.competences-content.level'), 276, y);
+  });
+
+  __fillCompetences({ parameters: [272, 295, 319], areaNumber: 0 });
+  __fillCompetences({ parameters: [375, 400, 424, 447], areaNumber: 1 });
+  __fillCompetences({ parameters: [504, 528, 552, 575], areaNumber: 2 });
+  __fillCompetences({ parameters: [632, 656, 679], areaNumber: 3 });
+  __fillCompetences({ parameters: [736, 759], areaNumber: 4 });
+
+  function __fillCompetences({ parameters, areaNumber }) {
+    parameters.forEach((y, index) => {
+      const resultCompetence = data.resultCompetenceTree.areas[areaNumber].resultCompetences[index];
+      pdf.fillColor('#253858').fillOpacity(1).text(resultCompetence.name, 65, y);
+
+      if (_isLevelShouldBeDisplayed(resultCompetence.level)) {
+        pdf.fillColor('#253858').fillOpacity(1).text(resultCompetence.level, 290, y);
+      } else {
+        pdf.rect(65, y, 200, 11).fillOpacity(0.5).fill('#ffffff');
+      }
+    });
+  }
+};
+
+function _isDeliveredAfterProfessionalizingStartDate({ isFrenchDomainExtension, deliveredAt, version }) {
+  return (
+    isFrenchDomainExtension &&
+    deliveredAt?.getTime() >= PROFESSIONALIZING_VALIDITY_START_DATE.getTime() &&
+    version === SESSIONS_VERSIONS.V2
```

Comment:
```suggestion
    version === AlgorithmEngineVersion.V2
```

Dans une PR en cours, on commence à supprimer l'utilisation des versions de session

Reply:
J'attend confirmation d'Agnès mais il se peut que je supprime carrément cette verif, ce sera encore mieux

Reply:
je le supprime complètement :) 
maintenant que les V3 utilisent leur propre méthode, plus besoin de vérifier qu'on est bien en V2